setup.py

```diff
@@ -19,7 +19,7 @@
     "cached-property",
     "entrypoints",
     "gitpython",
-    "python-gitlab",
+    "morefs[gitlab] @ git+https://github.com/skshetry/morefs@gitlabfs",
```

Comment:
Ok, I guess you need to merge this first https://github.com/iterative/morefs/pull/15, and then fix it like
```suggestion
    "morefs",
```
Could you please clarify the syntax here? Wonder how it works. It's going to be `pip install git+https://github.com/skshetry/morefs@gitlabfs`, but why `morefs[gitlab]` and why `[gitlab]` specifically?

Reply:
This is temporary, I'll merge that PR after all the tests pass here (testing gitlabfs and bitbucketfs is almost impossible).

> why morefs[gitlab] and why [gitlab] specifically?

`morefs` provides multiple filesystems, each with their own required dependencies. These are `extra-dependencies`, `morefs[gitlab]` will install required dependencies for gitlabfs (i.e. `python-gitlab`), and `morefs[bitbucket]` will install `requests` for example.

Reply:
Ah, got it! 😂 thanks for the patience in the explanation)